fixtures/pages-functions-with-routes-app/public/index.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html>
 	<body>
-		<h1>Bonjour de notre projet ✨pages-functions-with-routes-app✨!</h1>
+		<h1>
+			Bonjour de notre projet &#10024; pages-functions-with-routes-app &#10024;!
```

Comment:
Not to be a total nerd but you probably want `Bienvenue sur notre projet` 🙈

Reply:
awwww...best review of them all <3

Reply:
✅